### DISCUSSION TITLE
Feedback: Offline mode / Disable update check

### DISCUSSION THREAD

**Discussion by DrRevert** [MikeSchulze/gdUnit4#585] [Ideas]
Hi.
I think the automatic update check and any kind of pinging outside server should be an opt-in feature. Unless I'm mistaken the only option currently available is "Update Notification Enabled" which only allow to disable the notification as in the dialog box displayed at the beginning, but the plugin still pings the server.
From the user experience working on custom branch of the engine, I don't feel comfortable having a plugin to contact server outside my control while I'm working on my project without a way to disable such a functionality. Plugin should at least respect the editor setting "Connection/Network Mode" when it's set to "Offline".

**Comment by MikeSchulze** [comment]
You can disable the update notification and no outgoing connection will happen.
![image](https://github.com/user-attachments/assets/14b626ca-835f-4633-aba5-8ddb0ecd6270)

**Reply by DrRevert** [reply]
> You can disable the update notification and no outgoing connection will happen. ![image](https://private-user-images.githubusercontent.com/347037/371875664-14b626ca-835f-4633-aba5-8ddb0ecd6270.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjc2MzE5NTEsIm5iZiI6MTcyNzYzMTY1MSwicGF0aCI6Ii8zNDcwMzcvMzcxODc1NjY0LTE0YjYyNmNhLTgzNWYtNDYzMy1hYmE1LThkZGIwZWNkNjI3MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyOVQxNzQwNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNDM2MTM2MjRmMTg1Y2ZjYzgzNjRiMTRhYzU5ZDhlNWIyNjIwNjk1ZTQ5ZjU5NTZlMTU1MTU0MDM4ZjI1MzFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.57I-wxZzGcGGuWlK0_jpvGNqxDwXD5KDXvwTyRkCEqw)

Oh. Alright. From the description of the setting I thought it only disables the popup from showing up but still performs the update check in the background. Thanks.

**Reply by MikeSchulze** [reply]
No problem.
Feedback always welcome
